backend/tests/integration/mailing/test_api_seminars.py

```diff
@@ -3,14 +3,15 @@
 from mock import patch
 
 from inspirehep.mailing.api.seminars import send_seminar_confirmation_email
+from inspirehep.utils import get_inspirehep_url
 
 
 @patch("inspirehep.mailing.api.seminars.send_email")
 def test_send_seminar_confirmation_email(mock_send_email, base_app, db, es_clear):
     expected_recipient = "johndoe@gmail.com"
     seminar = {"control_number": "1234"}
     expected_subject = "Your seminar (1234) has been successfully submitted!"
-    host = current_app.config["SERVER_NAME"]
+    host = get_inspirehep_url()
```

Comment:
I would hardcode this

Reply:
because we don't send emails from qa?

Reply:
@michamos I left it like this, since I don't think it's important and we do have same for all other tickets/emails.